### DISCUSSION TITLE
unable to acess benifits

### DISCUSSION THREAD

**Discussion by Rajesh4434** [community/community#135401] [GitHub Education]
### Hi everyone,

my approved badge is showing in purple color still i am unable to use benifits

**Comment by davevad93** [comment]
Hi @Rajesh4434 ,

Go to the [student developer pack offers page](https://education.github.com/pack/offers), scroll down to the `All offers` section and claim one of the available offers.

If you're just interested in the free `Copilot` subscription, you can claim it from [here](https://education.github.com/pack/redeem/copilot-student) or by installing the `Copilot` extension in your preferred IDE (e.g. `VS Code`) and logging in with your GitHub account.

If after doing this you're still unable to access your benefits, you should contact support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your issue.
